src/main/java/spoon/reflect/declaration/ModifierKind.java

```diff
@@ -70,7 +78,8 @@
 	@Override
 	public String toString() {
 		if (lowercase == null) {
-			lowercase = name().toLowerCase(java.util.Locale.US);
+			lowercase = name().replace('_', '-') // NON_SEALED => non-sealed
```

Comment:
Should the lowercase name then be added to the other constants aswell? Then, the field could be final and toString really does not have any state change anymore. Otherwise, it's just an inconsistent mix of different patterns.

Reply:
Yes improving the complete enum and every constant is my suggestion.